src/EventStore.Projections.Core/Services/Processing/EmittedStream.cs

```diff
@@ -142,7 +144,9 @@ public class StreamMetadata {
 			if (ioDispatcher == null) throw new ArgumentNullException("ioDispatcher");
 			if (readyHandler == null) throw new ArgumentNullException("readyHandler");
 			_streamId = streamId;
+			_streamIdSize = Helper.UTF8NoBom.GetBytes(_streamId).Length;
```

Comment:
```suggestion
			_streamIdSize = Helper.UTF8NoBom.GetBytes(_streamId).Length;
			_streamIdSize = Helper.UTF8NoBom.GetByteCount(_streamId);
```

when we just want the length it might be cheaper to do it this way

Reply:
thanks, that's better :+1: